<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaneLeads</class>
 <widget class="QWidget" name="PaneLeads">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>819</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Leads</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="buttonImportLeads">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonImportLeadsInteraction">
       <property name="text">
        <string>Import emails with interaction</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonImportLeadsPurchase">
       <property name="toolTip">
        <string>Import email for which there was an interaction done</string>
       </property>
       <property name="text">
        <string>Import emails of customers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemoveLeads">
       <property name="toolTip">
        <string>Remove the selected leads</string>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFilter">
       <property name="text">
        <string>Filter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonFilterReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableViewCustomers">
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
